src/config/config.ts

```diff
@@ -83,8 +83,8 @@ export class Config implements IConfig {
   public arch!: ArchTypes
 
   public bin!: string
-  public binAliases?: string[]
-  public binPath?: string
+  public binAliases?: string[] | undefined
```

Comment:
should any of these be readonly?  conversely, do we want people mutating them all?

Reply:
Can't be readonly because all these props are assigned in the `.load` method. Typescript doesn't like that